app/support/DbAdapter.js

```diff
@@ -567,11 +573,23 @@ export class DbAdapter {
   }
 
   async fetchUsers(ids) {
+    if (_.isEmpty(ids)) {
+      return [];
+    }
     const uniqIds = _.uniq(ids);
-    const cachedUsers = await Promise.all(uniqIds.map(this.getCachedUserAttrs));
+    let cachedUsers;
+    if (this.cache.store.name === 'redis') {
+      const { client, done } = await promisify(this.cache.store.getClient)();
```

Comment:
1. it's better to run `promisify` in constructor, not on each request
2. `done()` is a part of non-promisified call, it is not exposed on promisified objects or am I missing anything?